lib/options-manager.js

```diff
@@ -21,6 +23,7 @@ const toAbsoluteGlob = require('to-absolute-glob');
 const stringify = require('json-stable-stringify-without-jsonify');
 const murmur = require('imurmurhash');
 const isPathInside = require('is-path-inside');
+const {normalizePackageName} = require('@eslint/eslintrc/lib/shared/naming');
```

Comment:
I don't think accessing this `@eslint/eslintrc/lib/shared/naming` is very future-proof. Can you open an issue and ask them to expose it publicly?

Reply:
I found the public API